Y/Yosys/build_tarballs.jl

```diff
@@ -3,20 +3,19 @@
 using BinaryBuilder, Pkg
 
 name = "Yosys"
-version = v"0.9.0"
+version = v"0.11.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/YosysHQ/yosys.git", "2e421feb0ea526468493ab008f3c72beb12c2bc6")
+    GitSource("https://github.com/YosysHQ/yosys.git", "360fed8e4d611fa725a4526cf960383b8e6c6e64")
 ]
 
 dependencies = [
     Dependency("boost_jll"; compat="=1.76.0"), # max gcc7
-    Dependency("Readline_jll"),
-    Dependency("Tcl_jll"),
-    Dependency("Zlib_jll"),
-    Dependency("ABC_jll"),
-    Dependency("Libffi_jll"; compat="~3.2.2"),
+    Dependency("Readline_jll"; compat="~8.1.1"),
+    Dependency("Tcl_jll"; compat="~8.6.11"),
+    Dependency("Zlib_jll"; compat="~1.2.11"),
```

Comment:
For Libffi the `~` compat is fine (well, necessary), but for the others I think it may be excessive
```suggestion
    Dependency("Readline_jll"; compat="8.1.1"),
    Dependency("Tcl_jll"; compat="8.6.11"),
    Dependency("Zlib_jll"; compat="1.2.11"),
```